fspiop/v2_0_ISO20022/openapi3/components/schemas/IBAN2007Identifier.yaml

```diff
@@ -8,5 +8,4 @@ description: >
   International Bank Account Number (IBAN)" and replaced by the more recent
   edition of the standard.
 type: string
-pattern: ^[A-Z]{2,2}[0-9]{2,2}[a-zA-Z0-9]{1,30}$
```

Comment:
This pattern is also used in the json scheme. Not sure why it need to be deleted.